src/ios/OAuthPlugin.swift

```diff
@@ -210,6 +219,10 @@ class OAuthPlugin : CDVPlugin, SFSafariViewControllerDelegate, ASWebAuthenticati
     }
 
 
+    @objc internal func _handleError_(_ notification : NSNotification) {
```

Comment:
This method doesn't need to end in an underscore, just `_handleError(_ notification : NSNotification)` is fine.

Yes, Swift has a lot of underscores in weird places 😛

Reply:
Yeah, I think i thought that I saw that ending underscore on another method and just copied it, but I guess that was just the underscore directly afterwards in the bracket 😅 

fixed.